### ISSUE TITLE
Send button disabled

### ISSUE THREAD

**Issue by larrykind** [globaleaks/GlobaLeaks#3171]
**Describe the bug**
Send button remains disabled at last step if mandatory fields are not filled within the questionnaire.

**To Reproduce**
1. set a questionnaire with at leat 1 mandatory field
2. fill the questionnaire but the mandatory one --> send turns to disabled
3. fill the mandatory field --> send button does not light on

**Expected behavior**

**Desktop (please complete the following information):**
 - OS: ubuntu
 - Browser: firefox 97
 - Globaleaks Version: 4.7.10

**Comment by evilaliv3** [comment]
Thank you for reporting this. This is actually an expected behavior but a message about the missing question should appear on top of the application.

Did it appear?

When you have in general questionnaire that can reproduce the issue please attach them so that we could immediately reproduce.

**Comment by larrykind** [comment]
@evilaliv3 yes, error appears, but once filled the mandatory field, send button doesn't unlock. Different behaviour for additional questionnaire, send button locks and unlocks correctly with mandatory fields (anyway my additional questionnaire is very small), works also with files.

**Comment by evilaliv3** [comment]
Thank you, i've identified the issue.

Thhe bug is due to the mandatory file upload and is related to the change that uploads the file only when the submission is completed.

Probably now we need a different condition for verifying that the user has added the file.

I will debug this furtherly and find out a solution.

**Comment by evilaliv3** [comment]
Corrected, thank you @larrykind 

The fix will be included in release 4.7.11
